matter_demos.xml

```diff
@@ -1,723 +1,1033 @@
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Thermostat Application for BRD4164A to be used with RS9116 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4163a.demo.thermostat_app_wifi_rs9116" label="Matter - Thermostat over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Thermostat app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4163a"/>
+		<property key="demos.imageFile" value="Examples/thermostat-app-wifi/rs9116/BRD4163A/chip-efr32-thermostat-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Thermostat Application for BRD4163A to be used with RS9116 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4186c.demo.thermostat_app_wifi_rs9116" label="Matter - Thermostat over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Thermostat app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4186c"/>
+		<property key="demos.imageFile" value="Examples/thermostat-app-wifi/rs9116/BRD4186C/chip-efr32-thermostat-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Thermostat Application for BRD4186C to be used with RS9116 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4187c.demo.thermostat_app_wifi_siwx917" label="Matter - Thermostat over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Thermostat app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4187c"/>
+		<property key="demos.imageFile" value="Examples/thermostat-app-wifi/siwx917/BRD4187C/chip-efr32-thermostat-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Thermostat Application for BRD4187C to be used with SIWX917 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4186c.demo.thermostat_app_wifi_siwx917" label="Matter - Thermostat over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Thermostat app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4186c"/>
+		<property key="demos.imageFile" value="Examples/thermostat-app-wifi/siwx917/BRD4186C/chip-efr32-thermostat-example.s37"/>
```

Comment:
I see that the path is different for the same app for Wi-Fi for SiWx917 and RS9116. The same binary is used for both. Am I right in understanding that our approach is to copy the same binary to these two different paths?